ex4_v4.py

```diff
@@ -0,0 +1,26 @@
+def compter_lignes(chemin_fichier) :
```

Comment:
The name of the parameter lead the reader to think that is should be string, but it is a file stream. You should either rename your parameter. or take a string as it suggests